packages/block-library/src/columns/style.scss

```diff
@@ -18,7 +18,9 @@
 
 	// Beyond mobile, allow 2 columns.
 	@include break-small() {
-		flex-basis: 50%;
+
+		// Size should be half the available width, minus the margins added below
```

Comment:
Needs a period. Hey, actually I think I'll stop these reviews and just push a fix. Hold on.